scripts/RenameFiles.bat

```diff
@@ -0,0 +1,185 @@
+echo                  ^|                                                               ^|
+echo                  ^|      [1] Make system restore point                            ^|
+echo                  ^|                                                               ^|
+echo                  ^|      [2] Proceed without creating restore point               ^|
+echo                  ^|      ___________________________________________________      ^|
+echo                  ^|                                                               ^|
+echo                  ^|      [Q] Exit Module                                          ^|
+echo                  ^|                                                               ^|
+echo                  ^|                                                               ^|
+echo                  ^|_______________________________________________________________^|
+echo:          
+choice /C:12Q /N /M ">                                            Select [1,2,Q]: "
+
+cls
+if errorlevel 3 (
+	goto exit
+)
+if errorlevel 2 goto:renameFiles
+if errorlevel 1 (
+	echo Creating system restore point, please be patient.
+	wmic /Namespace:\\root\default Path SystemRestore Call CreateRestorePoint "Before CCStopper Rename Adobe Files Script", 100, 12
+	goto renameFiles
+)
+
+:renameFiles
+Powershell -ExecutionPolicy RemoteSigned -File .\StopProcesses.ps1
+for %%a in (%files%) do ( 
+	setlocal EnableDelayedExpansion
+	set "_=%%a" & set renamed=!_:.exe=.exe.renamed!
+
+	for %%f in (%%a) do set name=%%~nxf
+	for %%f in (!renamed!) do set renamedName=%%~nxf
+	
+	if %targetExists% == true (
+		rename %%a "!renamedName!" >nul 2>&1
+	) else if %renamedExists% == true (
+		rename !renamed! "!name!" >nul 2>&1
+	)
+	setlocal DisableDelayedExpansion
+)
+goto restartAsk
+
+:restartAsk
+cls
+:: Thanks https://github.com/massgravel/Microsoft-Activation-Scripts for the UI
+echo:
+echo:
+echo                   _______________________________________________________________
+echo                  ^|                                                               ^| 
+echo                  ^|                                                               ^|
+echo                  ^|                            CCSTOPPER                          ^|
+echo                  ^|                         Made by eaaasun                       ^|
+echo                  ^|                        RenameFiles Module                     ^|
+echo                  ^|      ___________________________________________________      ^|
+echo                  ^|                                                               ^|
+echo                  ^|              Renaming adobe process files complete!           ^|
+echo                  ^|                                                               ^|
+echo                  ^|      The system needs to restart for changes to apply.        ^|
```

Comment:
There's no need for the system to restart as long as all adobe processes were stopped before changing file names

Reply:
I used the acrobat fix script as a base, and forgot to cut this part of the code out